fluidBuild.config.cjs

```diff
@@ -205,11 +205,13 @@ module.exports = {
 			],
 			"fluid-build-tasks-eslint": [
 				// eslint doesn't really depend on build. Doing so just slows down a package build.
+				"^packages/test/snapshots/package.json",
 				"^packages/test/test-utils/package.json",
 			],
 			"fluid-build-tasks-tsc": [
 				// TODO: AB#7460 fix tsconfig reference path match on Windows
 				"^packages/tools/devtools/devtools-view/package.json",
+				"^packages/test/snapshots/package.json",
```

Comment:
What is special about this package compared to others?
I know why tools/devtools-view has an exception, but doesn't seem like the referenced issue matches.

Reply:
I figured it out I think. The policy does not treat `tsc --project tsconfig.json` the same as `tsc --project ./tsconfig.json`. Will be fixed in upcoming PR.